cmd/simulation-server/main.go

```
@@ -32,79 +31,61 @@ func main() {
-		// Start HTTP health endpoint for Kubernetes probes
-		go func() {
-			http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
-				w.WriteHeader(http.StatusOK)
-				w.Write([]byte("healthy"))
+	log.Printf("gRPC server listening on port %s", *grpcPortPtr)
+
+	// Start HTTP health endpoint for Kubernetes probes
+	go func() {
+		http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte("healthy"))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)